ci/build38.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
+# Special build for python3.8 until numpy puts its own wheels up
+
```

Comment:
you should create a bare conda env here

Reply:
how come?

Reply:
3.8 should not be very different than a regular setup

Reply:
It has to be different until there are 3.8 packages (python especially, but also our dependencies).

I think Brock has about the minimal difference we can get right now.